app/models/user.rb

```diff
@@ -716,6 +716,14 @@ def self.current
     @current_user ||= User.anonymous
   end
 
+  def self.execute_as(user)
```

Comment:
E.g. here: https://github.com/opf/openproject/blob/108160ed88dc1d8e97e21d141f1d0a668c68a5d8/app/views/user_mailer/work_package_updated.html.erb#L35

The problem is that we parse the textile for mails too. And parsing textile requires the `User.current` to be set correctly (e.g. "does the user have the permission to expand the macro #123?").

The *good way™* would be to allow injecting a user into the text formatter. But I deemed that out-of-scope at the time I wrote the fix (greatly because of feasibility/time constraints).